presidio-analyzer/presidio_analyzer/predefined_recognizers/in_vehicle_registration_recognizer.py

```diff
@@ -0,0 +1,1582 @@
+            "GJ": gj_list,
+            "HP": hp_list,
+            "HR": hr_list,
+            "JH": jh_list,
+            "JK": jk_list,
+            "KA": ka_list,
+            "KL": kl_list,
+            "LA": la_list,
+            "LD": ld_list,
+            "MH": mh_list,
+            "ML": ml_list,
+            "MN": mn_list,
+            "MP": mp_list,
+            "MZ": mz_list,
+            "NL": nl_list,
+            "OD": od_list,
+            "OR": or_list,
+            "PB": pb_list,
+            "PY": py_list,
+            "RJ": rj_list,
+            "SK": sk_list,
+            "TN": tn_list,
+            "TR": tr_list,
+            "TS": ts_list,
+            "UK": uk_list,
+            "UP": up_list,
+            "WB": wb_list,
+        }
+
+        union_territories = ["AN", "CH", "DH", "DL", "JK", "LA", "LD", "PY"]
+        states = [
+            "AP",
+            "AR",
+            "AS",
+            "BR",
+            "CG",
+            "GA",
+            "GJ",
+            "HR",
+            "HP",
+            "JH",
+            "KA",
+            "KL",
+            "MP",
+            "MH",
+            "MN",
+            "ML",
+            "MZ",
+            "NL",
+            "OD",
+            "PB",
+            "RJ",
+            "SK",
+            "TN",
+            "TS",
+            "TR",
+            "UP",
+            "UK",
+            "WB",
+            "UT",
+        ]
+        old_union_territories = ["CT", "DN"]
+        old_states = ["UL", "OR", "UA"]
+        non_standard_state_or_ut = ["DD"]
+
+        foreign_mission_codes = [
+            84,
+            85,
+            89,
+            93,
+            94,
+            95,
+            97,
+            98,
+            99,
+            102,
+            104,
+            105,
+            106,
+            109,
+            111,
+            112,
+            113,
+            117,
+            119,
+            120,
+            121,
+            122,
+            123,
+            125,
+            126,
+            128,
+            133,
+            134,
+            135,
+            137,
+            141,
+            145,
+            147,
+            149,
+            152,
+            153,
+            155,
+            156,
+            157,
+            159,
+            160,
+        ]
+
+        # armed_forces_vehicle_types = [
+        #    'A', 'B', 'C', 'D', 'E', 'F', 'H', 'K', 'P', 'R', 'X']
+
+        two_factor_registration_prefix = []
+        two_factor_registration_prefix.extend(union_territories)
+        two_factor_registration_prefix.extend(states)
+        two_factor_registration_prefix.extend(old_states)
+        two_factor_registration_prefix.extend(old_union_territories)
+        two_factor_registration_prefix.extend(non_standard_state_or_ut)
+        first_two_char = sanitized_value[:2].upper()
+        dist_code: str = ""
+        diplomatic_vehicle_codes = ["CC", "CD", "UN"]
+
+        match first_two_char:
```

Comment:
This is super cool but unfortunately not supported in python < 3.10, while Presidio still supports 3.8

Reply:
oversight error, thanks for pointing it. reverted to compatible syntax